### DISCUSSION TITLE
好きな電機メーカーを教えてください（Please-tell-me-your-favorite-Electronics-company）

### DISCUSSION THREAD

**Discussion by ninkofu** [NAIST-SE/AttractingContributors#468] [General]
Please tell me your favorite Electronics company.
好きな電機メーカーを教えてください.

例）
・Lenovo
・Sony
・Mitsubishi Electric

↓my fork
https://github.com/7518085/AttractingContributors-Please-tell-me-your-favorite-Electronics-company

**Comment by tmymzkr** [comment]
I done pull request 
please help me
https://github.com/m-tomoya/AttractingContributors-movie
